events/mbed_events.h

```diff
@@ -30,6 +30,60 @@
 
 using namespace events;
 
+namespace mbed {
+
+/**
+ * Return a pointer to an EventQueue, on which normal tasks can be queued.
+ *
+ * All calls to this return the same EventQueue - it and its dispatch thread
+ * are created on the first call to this function. The dispatch thread
+ * runs at default priority (currently osPriorityNormal).
+ *
+ * The EventQueue returned may be used to call() Events, or to chain() other
+ * EventQueues so that they are run in the same context.
+ *
+ * Events (or chained EventQueues) executing on the normal event queue should
+ * normally take less than 10ms to execute, to avoid starving other users. As
+ * such, users can expect that event latency will typically be 10ms or less,
+ * but could occasionally be significantly higher if many events are queued.
+ *
+ * If an RTOS is not present or the configuration option
+ * `events.shared-normal-dispatch-from-application` is set to true, then this
+ * does not create a dedicated dispatch thread - instead the application is
+ * expected to run the EventQueue's dispatch, eg from main. This is necessary
+ * for the event loop to work without an RTOS, or an RTOS system can can save
+ * memory by reusing the main stack.
+ *
+ * @return pointer to normal event queue
+ */
+events::EventQueue *get_shared_normal_event_queue();
```

Comment:
If you are using a debugger you would definitely want the asserts off. On the flip side, it would be great if CI could fail changes cause timing violations. Since this could vary from target to target, it might not be feasible.